File: test_smtp.php
<?php
require 'config/mail.php';
require 'includes/PHPMailer/Exception.php';
require 'includes/PHPMailer/PHPMailer.php';
require 'includes/PHPMailer/SMTP.php';

use PHPMailer\PHPMailer\PHPMailer;

$mail = new PHPMailer(true);
try {
    $mail->SMTPDebug = 2; // Verbose debug output
    $mail->isSMTP();
    $mail->Host       = MAIL_HOST;
    $mail->SMTPAuth   = true;
    $mail->Username   = MAIL_USERNAME;
    $mail->Password   = MAIL_PASSWORD;
    $mail->SMTPSecure = MAIL_ENCRYPTION;
    $mail->Port       = MAIL_PORT;
    
    $mail->setFrom(MAIL_FROM_ADDRESS, MAIL_FROM_NAME);
    $mail->addAddress('aryan.work0009@gmail.com');
    
    $mail->isHTML(false);
    $mail->Subject = 'SMTP Test';
    $mail->Body    = 'Testing SMTP configuration.';
    
    $mail->send();
    echo "Message has been sent\n";
} catch (Exception $e) {
    echo "Message could not be sent. Mailer Error: {$mail->ErrorInfo}\n";
}

File: test_smtp.py
import smtplib
from email.message import EmailMessage

try:
    print("Connecting to smtp.gmail.com on port 587...")
    server = smtplib.SMTP('smtp.gmail.com', 587)
    server.set_debuglevel(1)
    server.starttls()
    print("Attempting to login...")
    server.login('zeal.saaes.r311@gmail.com', 'snpcokiyduwrfajt')
    
    msg = EmailMessage()
    msg.set_content("This is a test from Python.")
    msg['Subject'] = "Python SMTP Test"
    msg['From'] = 'zeal.saaes.r311@gmail.com'
    msg['To'] = 'aryan.work0009@gmail.com'
    
    server.send_message(msg)
    server.quit()
    print("Success: Email sent!")
except Exception as e:
    print(f"Error: {e}")
